### PR TITLE
PP 1116: Fix commands/server communication delays caused by Nagle's

### PR DESCRIPTION
**Pull request by praveenhandigol**
#### Issue-ID

* **[PP-1116](https://pbspro.atlassian.net/browse/PP-1116)**

#### Problem description
* commands/server communication due to Nagle's when replies from server are larger than 1024 bytes
#### Cause / Analysis
* When the TCP buffer sizes are just a little larger than 1024 bytes, Nagle's algorithm slows down the communication between the server and the commands

#### Solution description
* Disable Nagle's algorithm on both sides of the connection from clients/server.

#### Checklist:

- [ ] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
- [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [x] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
- [ ] I have added  **manual test(s) to the Jira ticket and explained why PTL is not appropriate** for this case.
- [x] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [x] I have attached **test logs to the Jira ticket** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

File: src/lib/Libnet/net_server.c [hunk 1]
```diff
@@ -605,6 +605,16 @@ accept_conn(int sd)
 	}
 #endif
 
+	/*
+	 * Disable Nagle's algorithm on this TCP connection to server.
+	 * Nagle's algorithm is hurting cmd-server communication.
+	 */
+	if (set_nodelay(newsock) == -1) {
+		(void)close(newsock);
```
Comment (vstumpf) [inline src/lib/Libnet/net_server.c]: Does this not require an `#ifdef` guard for windows/linux like above?

**Comment by scc138** [comment]
This PR should be closed.  Version 14.1.2 is released and this did not make it in time.

**Comment by mike0042** [comment]
Was there another PR for this in master?

**Comment by scc138** [comment]
@mike0042: https://github.com/PBSPro/pbspro/pull/153

**Comment by bhroam** [comment]
Closing pull request as per @scc138

**Pull request closed without merging**